AI/AIInterface.cpp

```diff
@@ -173,7 +173,7 @@ namespace AIInterface {
             // ownership to all, while remembering which planets this is done
             // to.
             universe.InhibitUniverseObjectSignals(true);
-            for (auto& planet : universe.Objects().FindObjects<Planet>()) {
+            for (auto& planet : universe.Objects().all<Planet>()) {
                  if (planet->Unowned()) {
                      unowned_planets.push_back(planet);
```

Comment:
not related to this pull request, but another case where reserving space before pushing into vector would be good...

Reply:
There are multiple location where universe object id vector is not nessecary, but the size of the MR is already big enough as it is.